spec.md

```diff
@@ -782,6 +782,7 @@ For success responses, the following fields are defined:
 | --- | --- | --- |
 | state* | string | Valid values are `in progress`, `succeeded`, and `failed`. While `"state": "in progress"`, the Platform SHOULD continue polling. A response with `"state": "succeeded"` or `"state": "failed"` MUST cause the Platform to cease polling. |
 | description | string | A user-facing message that can be used to tell the user details about the status of the operation. If present, MUST be a non-empty string. |
+| instance_corrupt | boolean | For failed update and deprovisioning operations, this field indicates whether the instance is still usable or not. If the value is `true`, the Service Instance MUST be considered corrupt and the Platform SHOULD NOT allow the creation of new bindings. If the value is `false`, the Service Instance is in an unmodified and usable state. The default is true. |
```

Comment:
Last sentence: When not specified, the default value is `true`.

Reply:
If we head down this path, it seems like we'd need to support returning this value in the sync case too, right?

Reply:
While I would like to have the spec be precise about the state of the resource upon failure, for backwards compatibility can we? I'm wondering if the last sentence should be:
    If not specified, the state of the resource is unspecified by this specification.

Reply:
The sync case is covered by HTTP status codes. 4xx means the instance is still usable, 5xx means the instance is corrupt.

Reply:
You are right with the last sentence. But from a practical point of view, it makes no difference. The platform must treat the instance as corrupt.

Reply:
ah I missed the correlation to the 400/500 response codes.  Makes sense. thanks